pip_audit/_dependency_source/pyproject.py

```diff
@@ -142,8 +142,8 @@ def fix(self, fix_version: ResolvedFixVersion) -> None:
             # Now dump the new edited TOML to the temporary file.
             toml.dump(pyproject_data, tmp)
 
-            # And replace the original `pyproject.toml` file.
-            os.replace(tmp.name, self.filename)
+        # And replace the original `pyproject.toml` file.
+        os.replace(tmp.name, self.filename)
```

Comment:
Looks like an oversight, it should also be split. 

I moved it out because I had several tests failing with: `PermissionError: [WinError 32] The process cannot access the file because it is being used by another process`. When the `os.replace` is called, the file is still open for `r+w` and Windows don't like it.